infra/cifuzz/cifuzz.py

```diff
@@ -79,6 +79,19 @@
     level=logging.DEBUG)
 
 
+def checkout_specified_commit(build_repo_manager, pr_ref, commit_sha):
+  """Checks out the specified commit or pull request using
+  build_repo_manager."""
+  try:
+    if pr_ref:
+      build_repo_manager.checkout_pr(pr_ref)
+    else:
+      build_repo_manager.checkout_commit(commit_sha)
+  except (RuntimeError, ValueError):
+    logging.error('Can not check out requested state %s.', pr_ref or commit_sha)
+    logging.error('Using current repo state.')
```

Comment:
nit: why two error messages ? can concatenate at end of last one.

Reply:
Done.